src/pages/scroll.ts

```diff
@@ -83,6 +83,63 @@ function copyHandler(values: Values) {
 }
 
 export function scrollGenerator() {
+  const previewElement = document.getElementById(
+    'scroll-preview'
+  ) as HTMLElement;
```

Comment:
We have a way of getting elements it should be in this file:
https://github.com/Dun-sin/Code-Magic/blob/main/src/lib/getElements.ts